oxia/options_base.go

```diff
@@ -80,3 +82,41 @@ func PartitionKey(partitionKey string) BaseOption {
 		partitionKey: &partitionKey,
 	}
 }
+
+// --------------------------------------------------------------------------------------------
+
+type ConsistencyOpt struct {
```

Comment:
I wouldn't leave this as a per-operation option. I'd prefer to have it as a client instance wide option.